src/app/core/rest/link-instance.service.ts

```diff
@@ -38,58 +39,26 @@ export class LinkInstanceService {
   }
 
   public createLinkInstance(linkInstance: LinkInstance): Observable<LinkInstance> {
-    const linkInstances = LocalStorage.get(this.webStorageKey()) || {};
-
-    linkInstance.id = String(Math.floor(Math.random() * 1000000000000000) + 1);
-    linkInstances[linkInstance.id] = linkInstance;
-
-    LocalStorage.set(this.webStorageKey(), linkInstances);
-
-    return Observable.of(linkInstance);
+    return this.httpClient.post<LinkInstance>(this.restApiPrefix(), linkInstance);
   }
 
   public updateLinkInstance(id: string, linkInstance: LinkInstance): Observable<LinkInstance> {
-    const linkInstances = LocalStorage.get(this.webStorageKey()) || {};
-
-    linkInstances[id] = linkInstance;
-
-    LocalStorage.set(this.webStorageKey(), linkInstances);
-
-    return Observable.of(linkInstance);
+    return this.httpClient.put<LinkInstance>(this.restApiPrefix(id), linkInstance);
   }
 
   public deleteLinkInstance(id: string): Observable<string> {
-    const linkInstances = LocalStorage.get(this.webStorageKey()) || {};
-
-    delete linkInstances[id];
-
-    LocalStorage.set(this.webStorageKey(), linkInstances);
-
-    return Observable.of(id);
+    return this.httpClient.delete(this.restApiPrefix(id))
+      .pipe(switchMap(() => Observable.of(id)));
```

Comment:
You can just use `map` operator here and you do not need to create a new observable.